hack/metrics/user-workload-monitoring-config.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    prometheus:
+      remoteWrite:
+      - url: https://prometheus-prod-10-prod-us-central-0.grafana.net/api/prom/push
```

Comment:
TODO: parameterize this

Reply:
@alvaroaleman what's your feeling on whether this stuff (and/or the scripts) should be ported to Go instead of shell/YAML? The moment I find myself needing to parameterize YAML (and through Bash) I start to think about whether it's time to move to Go, but that could also be a followup refactor... (cc @csrwng)